readthedocs.yml

```diff
@@ -0,0 +1,6 @@
+# http://blog.readthedocs.com/python-36-support/
+build:
+    image: latest
```

Comment:
README.md points to `stable`, but this builds `latest` - is stable tied to the `master` branch?

Reply:
^this references the readthedocs machine image, not our projects version